UKHomeOffice/cop-ui
# COP-9604 Clear filter not working as expected on COP targets page

## Description
This PR fixes a bug where the clear filter button did not reset target count on the first try, required two tries to reset the target count (COP-9604).

* JIRA Ticket: https://support.cop.homeoffice.gov.uk/browse/COP-9604

## To Test
- Pull and run against dev
- On cop-ui, go to targets (**'Targets assigned to your teams'** box)
- Note down the **total targets** displayed
- Apply more than 1 filter and check the number of targets
- Clear all filters
- Check the total targets displayed

## Developer Checklist

\* Required
